lib/find-view-matches.js

```diff
@@ -57,6 +63,10 @@ function attrSelector(attr, value) {
 	].join('');
 }
 
+function prefixSelector(prefix, value) {
+	return value.replace(new RegExp('^\\'+ prefix +'?(.*)$', 'i'), function (match) { return prefix + match });
```

Comment:
I got now what you wanted to achieve, and I must say, the solution in the comment looks more readable to me :)